### PR TITLE
Resolved Cocoapod linting issues

### PR DESCRIPTION
**Pull request by codytwinton**
Thanks for proposing a pull request.

To help us review the request, please complete the following:

- [ ] sign [contributor license agreement](https://developers.facebook.com/opensource/cla)
- [ ] describe the change (for example, what happens before the change, and after the change)

### CODE REVIEW

**Comment by robtimp** [comment]
It’s trying to run the lint on the 5.0 rc version, for some reason. I assume we want to lint the latest 4.x release?

**Comment by codytwinton** [comment]
@robtimp - Nope, the latest release is `5.0.0-rc.1`. This was on purpose.

**Comment by robtimp** [comment]
@codytwinton The pod specs need to be updated with the new tvOS 10.0 deployment target, to match the config files.

File: Configurations/FacebookSDK-Project-TVOS.xcconfig [hunk 1]
```diff
@@ -21,7 +21,7 @@
 #include "FacebookSDK-Library.xcconfig"
 
 ARCHS = x86_64 arm64
-TVOS_DEPLOYMENT_TARGET = 9.0
+TVOS_DEPLOYMENT_TARGET = 10.0
```
Comment (robtimp) [inline Configurations/FacebookSDK-Project-TVOS.xcconfig]: We should update the podspec to match

File: Configurations/Platform/tvOS.xcconfig [hunk 2]
```diff
@@ -17,7 +17,7 @@
 // CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 SDKROOT = appletvos
-TVOS_DEPLOYMENT_TARGET = 9.0
+TVOS_DEPLOYMENT_TARGET = 10.0
```
Comment (robtimp) [inline Configurations/Platform/tvOS.xcconfig]: Likewise

File: .travis.yml [hunk 3]
```diff
@@ -50,7 +50,7 @@ jobs:
     - stage: test
       name: Pod Lint - Xcode 10.2
       osx_image: xcode10.2
-      script: sh scripts/run.sh lint cocoapods --allow-warnings
+      script: sh scripts/run.sh lint cocoapods
```
Comment (robtimp) [inline .travis.yml]: Won't the warnings about missing `info.plist` files cause this to fail?

**Pull request closed without merging**